tests/mockedObjects.ts

```diff
@@ -0,0 +1,255 @@
+  uuid: '730bfa24-eb57-11ec-b722-0242ac150002',
+  createdAt: new Date('2022-05-27T15:18:42.000Z'),
+  updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+  addresses: [
+    {
+      id: 1,
+      address: 'mockedaddress0nkus8hzv367za28j900c7tv5v8pc',
+      createdAt: new Date('2022-05-27T15:18:42.000Z'),
+      updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+      chainId: 1,
+      paybuttonId: 1
+    },
+    {
+      id: 2,
+      address: 'mockedaddress0nkush83z76az28900c7tj5vpc8f',
+      createdAt: new Date('2022-05-27T15:18:42.000Z'),
+      updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+      chainId: 2,
+      paybuttonId: 1
+    }
+  ]
+}
+
+export const mockedPaybuttonAddress = {
+  id: 1,
+  address: 'qpuzcemdv2hhng4fpyjxzfp0axsyltrghutla9rfnm',
+  createdAt: new Date('2022-05-27T15:18:42.000Z'),
+  updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+  chainId: 1,
+  paybuttonId: 1
+}
+
+export const mockedPaybuttonList = [
+  {
+    id: 1,
+    providerUserId: 'mocked-uid',
+    name: 'mocked-name-1',
+    buttonData: 'mockedData',
+    uuid: '730bfa24-eb57-11ec-b722-0242ac150002',
+    createdAt: new Date('2022-05-27T15:18:42.000Z'),
+    updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+    addresses: [
+      {
+        id: 1,
+        address: 'mockedaddress0nkus8hzv367za28j900c7tv5v8pc',
+        createdAt: new Date('2022-05-27T15:18:42.000Z'),
+        updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+        chainId: 1,
+        paybuttonId: 1
+      },
+      {
+        id: 2,
+        address: 'mockedaddress0nkush83z76az28900c7tj5vpc8f',
+        createdAt: new Date('2022-05-27T15:18:42.000Z'),
+        updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+        chainId: 2,
+        paybuttonId: 1
+      }
+    ]
+  },
+  {
+    id: 2,
+    providerUserId: 'mocked-uid',
+    name: 'mocked-name-2',
+    buttonData: 'mockedData',
+    uuid: '133fb8aa-eb57-11ec-b722-0242ac150002',
+    createdAt: new Date('2022-05-27T15:18:42.000Z'),
+    updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+    addresses: [
+      {
+        id: 3,
+        address: 'mockedaddress0nkus8hzv367za28j900c7tv5v8pc',
+        createdAt: new Date('2022-05-27T15:18:42.000Z'),
+        updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+        chainId: 1,
+        paybuttonId: 2
+      },
+      {
+        id: 4,
+        address: 'mockedaddress0nkush83z76az28900c7tj5vpc8f',
+        createdAt: new Date('2022-05-27T15:18:42.000Z'),
+        updatedAt: new Date('2022-05-27T15:18:42.000Z'),
+        chainId: 2,
+        paybuttonId: 2
+      }
+    ]
+  }
+]
+
+// Chain
+export const mockedChain = {
+  id: 1,
+  slug: 'bitcoincash',
+  ticker: 'bch',
+  title: 'Bitcoin Cash',
+  createdAt: new Date('2022-05-27T15:18:42.000Z'),
+  updatedAt: new Date('2022-05-27T15:18:42.000Z')
+}
+
+// Transaction
+export const mockedTransaction = {
+  id: 1,
+  hash: 'Yh5DRDjd3AarAvQA1nwpPI4daDihY6hQfnMV6UKFqZc=',
+  paybuttonAddressId: 1,
+  amount: '431247724',
```

Comment:
Assuming we change to Decimal(16,2), this will need to change to `431247724` or `431247724.00`.